### PR TITLE
[WiP] All docstrings + a few improvements

### LINKED ISSUE #53: Docstring implementation
### Summary

I need to add Docstrings to methods.

### The Problem

Documentation is hard.

### The Ideal Solution

To add Docstrings.

### The Current Solution

_No response_

### Checklist

- [x] This bug was found on the latest version of EpikCord
- [X] Full details of the requested feature have been provided/shown

### Additional Context

I added the "stopping library from progressing" label because it stops the library from becoming as accessible as it could be.

### PR DESCRIPTION
**Pull request by nav-github01001**
## Description
this PR closes #53 

## Checklist

- [ ] - All code changes have been tested, if any
- [ ] - The documentation has been updated accordingly for any code changes
- [ ] - This PR fixes an already existing issue
- [ ] - This PR adds a new feature (new functions, methods, classes, parameters, etc.)
- [ ] - This PR fixes a bug
- [ ] - This PR is a breaking change (removes/renames functions, parameters etc.)
- [ ] - This PR is NOT a code change (changes Documentation files, README.md, CONTRIBUTING.md etc.)

### CODE REVIEW

**Comment by nav-github01001** [comment]
feel free to commit here

**Comment by nav-github01001** [comment]
i need help

**Comment by Sigmanificient** [comment]
You forget to tick "This pr is NOT a code change"

**Comment by nav-github01001** [comment]
It is sort of also a code change, must update PR name

File: EpikCord/__init__.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
 
 from aiohttp import ClientSession, ClientResponse, ClientWebSocketResponse
 
-from .__main__ import __version__
+__version__ = "0.5.2"
```
Comment (Sigmanificient) [inline EpikCord/__init__.py:39]: Dont put `__version__` in `__main__` smh, put it in `__init__`. The `__main__` is the package entry point not something you should store importables in.

File: EpikCord/__init__.py [hunk 2]
```diff
@@ -451,13 +451,42 @@ def __init__(self, client, data: dict):
...
+        *The id and the emoji name can be obtained by entering a backslash ahead of an emoji ``\\📃``
+        """
         emoji = _quote(emoji)
         response = await self.client.http.put(
             f"channels/{self.channel_id}/messages/{self.id}/reactions/{emoji}/@me"
         )
         return await response.json()
 
-    async def remove_reaction(self, emoji: str, user=None):
+    async def remove_reaction(self, emoji: str, user:User=None):
```
Comment (Sigmanificient) [inline EpikCord/__init__.py:474]:
```suggestion
    async def remove_reaction(self, emoji: str, user: Optional[User] = None):
```

File: EpikCord/__init__.py [hunk 3]
```diff
@@ -610,6 +718,17 @@ async def fetch_messages(
         after: Optional[str] = None,
         limit: Optional[int] = None,
     ) -> List[Message]:
+        """Fetches a list of messages from the parent channel or DM
+
+        Args:
+            around (Optional[str], optional): Get messages around this message ID (before and after the message was sent). Defaults to None.
+            before (Optional[str], optional): Get messages before this message ID. Defaults to None.
+            after (Optional[str], optional): Get messages after this message ID. Defaults to None.
+            limit (Optional[int], optional): Number of messages to return. Defaults to 50.
```
Comment (Sigmanificient) [inline EpikCord/__init__.py:727]: Wtf `(Optional[...], optional)`. I think that `Optional[str]` is already clear enough lol

**Review by Sigmanificient — CHANGES_REQUESTED** [review]

File: EpikCord/__init__.py (hunk 2 above)
Comment (TheUntraceable) [reply EpikCord/__init__.py:474]: Should be Union[GuildMember, User]

File: EpikCord/__init__.py (hunk 3 above)
Comment (TheUntraceable) [reply EpikCord/__init__.py:727]: Yeah...

File: EpikCord/__init__.py (hunk 2 above)
Comment (Sigmanificient) [reply EpikCord/__init__.py:474]: maybe `Optional[GuildMember]` then cause `GuildMember` inherit from `User`... right?

Comment (TheUntraceable) [reply EpikCord/__init__.py:474]: Yeah, you can't remove reactions in DM's, the only place you get `User`, so yeah makes sense.

File: EpikCord/__init__.py (hunk 3 above)
Comment (nav-github01001) [reply EpikCord/__init__.py:727]: ... I got the template from the VSCode extension Sphinx itself recommends https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring

Comment (TheUntraceable) [reply EpikCord/__init__.py:727]: Remove it

**Pull request closed without merging**